src/Utility/DimensionUtility.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Utility;
+
+/**
+ * A class for dealing with Dimensions.
+ *
+ * @since x.x.x
+ */
+class DimensionUtility {
+
+	/**
+	 * Width.
+	 *
+	 * @var int
+	 */
+	public int $x;
+	/**
+	 * Height.
+	 *
+	 * @var int
+	 */
+	public int $y;
+
+	/**
+	 * DimensionUtility constructor.
+	 *
+	 * @param int $x Width.
+	 * @param int $y Height.
+	 */
+	public function __construct( int $x, int $y ) {
+		$this->x = $x;
+		$this->y = $y;
+	}
+
+	/**
+	 * Checks if the dimension is bigger or equal than the other one.
+	 *
+	 * @param DimensionUtility $target The dimension to be compared.
+	 *
+	 * @return bool true if the dimension is bigger or equal than the other one otherwise false.
+	 */
+	public function is_bigger( DimensionUtility $target ): bool {
+		return $this->x >= $target->x && $this->y >= $target->y;
+	}
```

Comment:
💅  Thinking about this again. I wonder if we should omit `or equals` on the `is_bigger` check. WDTY?

Reply:
I added `or equals` because the condition is `>=`  or we could rename this function to `is_bigger_or_equal` if it is a bit confusing. WDYT?

Reply:
We are checking if it meets the required minimum size, so what about `is_minimum_size`

Reply:
@mikkamp I missed this comment yesterday, yes that it is a more convenient name. See [da76e8e](https://github.com/woocommerce/google-listings-and-ads/pull/1804/commits/da76e8e6d4018b64a3ef17264ede41604f224999)